.github/workflows/daily.yaml

```diff
@@ -199,13 +199,14 @@ jobs:
         shell: bash
         run: tar -xzf maven-repo.tgz -C ~
       - name: Install cl.exe
-        uses: ilammy/msvc-dev-cmd@v1
-      - uses: microsoft/setup-msbuild@v1
+        uses: ilammy/msvc-dev-cmd@v1.12.0
+      - uses: microsoft/setup-msbuild@v1.1.3
```

Comment:
Please see https://github.com/microsoft/setup-msbuild/tags, both ilammy/msvc-dev-cmd@v1 and microsoft/setup-msbuild@v1.1.3 are tags for https://github.com/microsoft/setup-msbuild/commit/34cfbaee7f672c76950673338facd8a73f637506. Why is this change needed?

Reply:
same reasoning as before, but for this GHA.

Reply:
I say you are actually not changing anything, just saying we should stick with some minor version. Both tags target same commit.

Reply:
![image](https://user-images.githubusercontent.com/43821672/203369192-0b0f9759-e32e-4d34-8fed-2fd67cb8191d.png)

Reply:
thank you @michalvavrik, didn't spot it.
Done